setup.py

```diff
@@ -1,6 +1,16 @@
 from setuptools import setup, find_packages
 
 install_req = [
+    'oda-knowledge-base',
```

Comment:
do we still need this?

Reply:
In this PR I make changes backward-compatible (at least I hope they are) 
As you currently store info about installed workflows in the central KG, the bot can continue to use the lib. This is configurable